routers/web/user/profile.go

```diff
@@ -91,6 +92,38 @@ func Profile(ctx *context.Context) {
 		ctx.Data["RenderedDescription"] = content
 	}
 
+	repo, err := repo_model.GetRepositoryByName(ctx.ContextUser.ID, ".profile")
+	if err == nil && !repo.IsEmpty {
+		gitRepo, err := git.OpenRepository(ctx, repo.RepoPath())
```

Comment:
The code looks like duplicated.

Reply:
It's present both in the ```header.go``` and ```profile. Go``` files. I can refactor the header version to only check for the existence of the file versus simply checking for a returned output

Reply:
Reworked to specifically check for existence in ```header.go``` instead of requesting entire file.